<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgSelectionMesure</class>
 <widget class="QWidget" name="WdgSelectionMesure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>48</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelNoEchantillon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>échantillon n°</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderEchantillon">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>461</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelNoMinEchantillon">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxEchantillon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelNoMaxEchantillon">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>0</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderEchantillon</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxEchantillon</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxEchantillon</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderEchantillon</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
